nvflare/app_common/launchers/subprocess_launcher.py

```diff
@@ -58,28 +46,23 @@ def launch_task(self, task_name: str, shareable: Shareable, fl_ctx: FLContext, a
             command = self._script
             env = os.environ.copy()
             command_seq = shlex.split(command)
-            log_file = os.path.join(self._app_dir, f"{task_name}_logfile")
 
             self._process = subprocess.Popen(
                 command_seq,
-                stdout=subprocess.PIPE,
                 stderr=subprocess.STDOUT,
                 cwd=self._app_dir,
                 env=env,
             )
-            self._log_thread = Thread(target=log_subprocess_output, args=(self._process, log_file))
-            self._log_thread.start()
             return True
         return False
 
-    def wait_task(self, task_name: str, fl_ctx: FLContext, timeout: Optional[float] = None) -> LauncherCompleteStatus:
+    def wait_task(self, task_name: str, fl_ctx: FLContext, timeout: Optional[float] = None) -> LauncherRunStatus:
```

Comment:
Since LauncherRunStatus is no longer enum, the return type here should be str.

Reply:
thanks, fixed.